source/navitiacommon/navitiacommon/bin.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2001-2014, Canal TP and/or its affiliates. All rights reserved.
```

Comment:
`bin.py` is quite confusing without any context as a `bin` has nothing to do with a `binary`.

Reply:
Sold for `process.py`